scvi/benchmark.py

```diff
@@ -63,33 +42,7 @@ def ldvae_benchmark(dataset, n_epochs, use_cuda=True):
     return trainer
 
 
-def all_benchmarks(n_epochs=250, use_cuda=True, save_path='data/', show_plot=True):
-    cortex_benchmark(n_epochs=n_epochs, use_cuda=use_cuda, save_path=save_path, show_plot=show_plot)
-
-    harmonization_benchmarks(n_epochs=n_epochs, use_cuda=use_cuda, save_path=save_path)
-    annotation_benchmarks(n_epochs=n_epochs, use_cuda=use_cuda, save_path=save_path)
-
-
-def benchmark_fish_scrna(gene_dataset_seq, gene_dataset_fish):
-    gene_names = gene_dataset_fish.gene_names
-    indexes_to_keep = np.arange(len(gene_names))
-    vae = VAEF(gene_dataset_seq.nb_genes, indexes_to_keep, n_layers_decoder=2, n_latent=6,
-               n_layers=2, n_hidden=256, reconstruction_loss='nb', dropout_rate=0.3, n_labels=7, n_batch=2,
-               model_library=False)
-    trainer = TrainerFish(vae, gene_dataset_seq, gene_dataset_fish, train_size=0.9,
-                          frequency=5, weight_decay=0.35, n_epochs_even=100, n_epochs_kl=1000,
-                          cl_ratio=0, n_epochs_cl=100)
-    trainer.train(n_epochs=1, lr=0.0008)
-    concatenated_matrix = np.concatenate(
-        (gene_dataset_fish.X[:, vae.indexes_to_keep], gene_dataset_seq.X[:, vae.indexes_to_keep]))
-    concatenated_matrix = np.log(1 + concatenated_matrix)
-    pca = PCA(n_components=9)
-    latent_pca = pca.fit_transform(concatenated_matrix)
-    pca_latent_fish = latent_pca[:gene_dataset_fish.X.shape[0], :]
-    pca_latent_seq = latent_pca[gene_dataset_fish.X.shape[0]:, :]
-    pca_values_seq = gene_dataset_seq.X
-    pca_labels_seq = gene_dataset_seq.labels
-    pca_labels_fish = gene_dataset_fish.labels
-    _ = proximity_imputation(pca_latent_seq, pca_values_seq[:, 0], pca_latent_fish, k=5)
-    _, _, = compute_accuracy_nn(pca_latent_seq, pca_labels_seq.ravel(), pca_latent_fish,
-                                pca_labels_fish.ravel())
+def all_benchmarks(n_epochs=250, use_cuda=True, save_path="data/", show_plot=True):
+    cortex_benchmark(
+        n_epochs=n_epochs, use_cuda=use_cuda, save_path=save_path, show_plot=show_plot
+    )
```

Comment:
Could we delete the whole `benchmark.py` file, and the unit tests that call any of the remain benchmarks here? We never really developed a complete set of benchmarks. Seems like we've converged on using notebooks for everything -- for better or worse.

Reply:
I'm having a look to see if the tests covered by `benchmark.py` are redundant or can be moved somewhere else.

Reply:
I removed `benchmark.py`, though I found a mention of the file in the auto-generated docs. https://github.com/YosefLab/scVI/blob/48740851f32e1bee8d4e87f3b18a2bec9ec97e9c/docs/scvi.rst#L16
@jeff-regier , do you know how to update these files?

Reply:
`make docs` does it I think